ResearchKit/Common/ORKFormStep.m

```diff
@@ -139,6 +139,18 @@ - (void)setFormItems:(NSArray<ORKFormItem *> *)formItems {
 
 @implementation ORKFormItem
 
+- (instancetype)initWithIdentifier:(NSString *)identifier text:(NSString *)text optional:(BOOL) optional answerFormat:(ORKAnswerFormat *)answerFormat {
+    self = [super init];
+    if (self) {
+        ORKThrowInvalidArgumentExceptionIfNil(identifier);
+        _optional = optional;
+        _identifier = [identifier copy];
+        _text = [text copy];
+        _answerFormat = [answerFormat copy];
+    }
+    return self;
+}
+
 - (instancetype)initWithIdentifier:(NSString *)identifier text:(NSString *)text answerFormat:(ORKAnswerFormat *)answerFormat {
```

Comment:
We might want to reuse your new initializer for this initializer with the default value of `YES` for optional parameter.